Package.swift

```diff
@@ -1,18 +1,27 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.1
 
 import PackageDescription
 
 let package = Package(
     name: "Imperial",
+    platforms: [
+       .macOS(.v10_14)
```

Comment:
This will need to be 10.15 if we're relying on JWT

Reply:
done